package.json

```diff
@@ -155,19 +155,21 @@
     ]
   },
   "scripts": {
-    "compile": "tsc -noEmit && node .esbuild.config.js",
+    "compile": "tsc --noEmit && node .esbuild.config.js",
     "lint": "eslint src --ext ts",
     "watch": "node .esbuild.config.js --watch",
-    "test": "node ./src/test/runTest.js"
+    "test": "tsc --noEmit && node ./src/test/runTest.js"
   },
   "devDependencies": {
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.18.6",
     "@linaria/esbuild": "^4.1.4",
     "@linaria/shaker": "^4.2.1",
+    "@types/babel__core": "^7.1.20",
```

Comment:
These were needed since Linaria's types reference them

Reply:
I wonder why linaria doesn't have it as a dev dependency then? I guess types cannot have dev dependencies

Reply:
It's always been kind of an annoying thing: you have types that expose stuff from another package, but you also don't want to mark that package as a 'production dependency' since it might not actually be used at runtime and therefore unnecessary for non-TypeScript consumers.